generators/rest-adapter-generator/src/test/java/uk/gov/justice/config/GeneratorPropertiesFactory.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.justice.config;
+
+import static uk.gov.justice.services.test.utils.core.reflection.ReflectionUtil.setField;
+
+import uk.gov.justice.maven.generator.io.files.parser.core.GeneratorProperties;
+import uk.gov.justice.services.generators.commons.config.CommonGeneratorProperties;
+
+public class GeneratorPropertiesFactory {
+
```

Comment:
A lot of copies of GeneratoryPropertiesFactory. If it's useful, maybe promote it to a class in the test-utils module. Maybe rename to builder?